src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -91,45 +90,55 @@ public override NodeInputData InputData
         /// </summary>
         public int SelectedIndex
         {
-            get { return selectedIndex; }
+            get => selectedIndex;
             set
             {
                 //do not allow selected index to
                 //go out of range of the items collection
                 if (value > Items.Count - 1 || value < 0)
                 {
+                    Warning(Dynamo.Properties.Resources.NothingIsSelectedWarning);
```

Comment:
hmm please give it some testing, I think the default index is -1, so adding the warning here may affect default behavior of this Dynamo node, e.g. when user place it, it display a warning already?

Reply:
If your testing is good, then it is probably safe to do so, I am not 100% sure :)

Reply:
OK, thanks for noticing this. Adding it to the setter isn't good, as `SelectedIndex` is set to -1 when the dropdown is opened. I added the warning back into `UpdateValueCore`.